README.md

```diff
@@ -2164,14 +2149,14 @@ Here is the equivalence of the stdlib `reject` function:
 
 Reverses the order of a string or array.
 
-*Note*: The same can be done with the built-in [`reverse_each`](https://docs.puppet.com/puppet/latest/function.html#reverse_each) function in Puppet.
+> *Note*: The same can be done with the built-in [`reverse_each`](https://docs.puppet.com/puppet/latest/function.html#reverse_each) function in Puppet.
 
 
 #### `round`
 
 **Deprecated:** This function has been replaced with a built-in [`round`](https://puppet.com/docs/puppet/latest/function.html#round) function as of Puppet 6.0.0.
 
-Rounds a number to the nearest integer
+It rounds a number to the nearest integer.
```

Comment:
Suggest not changing this because the original form is how function documentation is typically written in most programming languages. It answers the question "what does it do", and there is no need to start with "it". Typically it is something like "Performs...", "Computes...", "Calculates...", here it works to say just "Rounds..."